gigascience/gigadb-website
# Add top margin to table footer in dataset page

# Pull request for issue: #1843

- added margin top to screen widths under 1200px to prevent scrollbar from overlapping footer content

## How to test?

* Go to http://gigadb.gigasciencejournal.com/dataset/100006
* set files per page to 5
* change screen width
* verify that top margin of taable footer increases for small screens
* Ideally test in user reported setup (I couldn't see the issue to begin with in a remote desktop environment with such setup):

OS: Mac ventura 13.4
Browser: safari
Version 16.5

## How have functionalities been implemented?

Updated `less/current/modules/tables.less` to include `margin-top`

## Any issues with implementation?

* bootstrap sets breakpoints as variables, it would be better to use the bootstrap variables themselves for consistency, particularly if the site is made responsive eventually. I believe this cannot be done in the current setup because the less files are first compiled and then linked to the layout along with bootstrap minified css

## Any changes to automated tests?

--
